.github/workflows/build.yml

```
@@ -0,0 +1,153 @@
+
+      - name: Build eltord
+        working-directory: /home/runner/code/eltord
+        run: |
+          if [ "${{ matrix.platform.use_cross }}" = "true" ]; then
+            cross build --release --verbose --target ${{ matrix.platform.target }}
+          else
+            cargo build --release --verbose --target ${{ matrix.platform.target }}
+          fi
+
+      - name: Rename binary
+        run: |
```

actionlint (1.7.7)

got unexpected character ' ' while lexing || operator, expecting '|'

(expression)